services/publicapi/service.go

```diff
@@ -28,8 +28,8 @@ func NewPublicApi(
 }
 
 func (s *service) SendTransaction(input *services.SendTransactionInput) (*services.SendTransactionOutput, error) {
-	s.reporting.Info("enter_send_transaction")
-	defer s.reporting.Info("exit_send_transaction")
+	s.reporting.Info("enter SendTransaction")
+	defer s.reporting.Info("exit SendTransaction")
```

Comment:
feels too verbose, but not sure its important to change this now (the change is simple :) )
not sure what kind of context this gives to whoever will read these messages

Reply:
Yes, it's not my errors, I just refactored them.. I assume they will disappear eventually